bluebottle/initiatives/tests/test_api.py

```diff
@@ -1552,3 +1553,66 @@ def test_submitted_initiatives_dont_show_on_map(self):
 
         data = response.json()
         self.assertTrue(len(data), 5)
+
+    def test_locations(self):
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(23.6851594, 43.0579025))
+        )
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(27.6851594, 34.0579025))
+        )
+
+        response = self.client.get(
+            self.url
+        )
+        data = response.json()
+
+        lat = data[0]['position'][0]
+        long = data[0]['position'][1]
+        lat_2 = data[1]['position'][0]
+        long_2 = data[1]['position'][1]
+        self.assertTrue(lat in data[0]['position'])
+        self.assertTrue(long in data[0]['position'])
+        self.assertTrue(lat_2 in data[1]['position'])
+        self.assertTrue(long_2 in data[1]['position'])
+
```

Comment:
You're doing this:
```
fruit = 'apple'
self.assertTrue(fruit in  ['apple'])
```
You probably want to check that `lat` or `data[0]['position'][0]` has a specific value., like
```
self.assertEqual(lat, '43.0579025')
```